CODEOWNERS.txt

```diff
@@ -0,0 +1,58 @@
+# Despite the name of this file, the people listed here DO NOT OWN the code in astropy subpackages,
+# the copyright is held by the project as a whole - see the license file for details. 
+# 
+# However, github currently (2022) does not offer a mechanism on their website to subscribe to
+# a fraction of newly open PRs in astropy. Either one chooses the "watch" the entire repository and 
+# receives  a large number of emails or one needs to manually check is new PRs for a certain sub-package
+# exist. This file is a workaround for that problem:
+# All names listed here by sub-packages will be requested for review for any new PR 
+# and thus get an email notification.
+# In many cases, the names below will be active sub-package maintainers, but anyone interested
+# can open a PR to add their github handle to any subpackage they are interested in.
+
+
+# Add your github username to opt-in to automatic review requests for
+# specific sub-packages here:
+
+# astropy/constants
+# astropy/convolution
+# astropy/coordinates
+astropy/cosmology           @nstarman
```

Comment:
```suggestion
astropy/cosmology           @astropy/cosmology
```

Reply:
Maybe other groups of sub-package maintainers should also form teams like this and as a unit decide if they want notifications like this?

Reply:
I personally think it's a good idea, but pragmatically it hardly makes a difference since most teams are only a few people. Teams have more longevity than individual maintainers, and there are nice options like review load balancing, but again, not much practical difference.